core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -51,6 +60,16 @@ static DeflaterOutputStream getEncodingOutputStream(HttpEncodingType encodingTyp
                 }
             case DEFLATE:
                 return new DeflaterOutputStream(out, true);
+            case BROTLI:
+                try {
+                    // We use 4 as the default level because it would save more bytes
+                    // than GZIP's default setting and compress data faster.
+                    // See: https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html
+                    return new BrotliOutputStream(out, new Encoder.Parameters().setQuality(4));
```

Comment:
What I meant was makes `Encoder.Parameters` singleton. We don't have to create a parameter every time if we use the same option.
```
private static final Encoder.Parameters BROTLI_PARAMETERS = new Encoder.Parameters().setQuality(4);

...
return new BrotliOutputStream(out, BROTLI_PARAMETERS);
```

Reply:
Thank you for comment! I followed the suggestion. Can you check it again? :grinning: